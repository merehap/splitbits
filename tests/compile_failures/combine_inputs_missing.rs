use splitbits::*;

fn main() {
    let a: u8 = 0b1010_0101;
    let result = combinebits!("aaaa aaaa bbbb bbbb");
    assert_eq!(result, 0b1010_0101_0000_1111u16);
}
