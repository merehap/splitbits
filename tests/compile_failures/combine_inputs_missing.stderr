error[E0425]: cannot find value `b` in this scope
 --> tests/compile_failures/combine_inputs_missing.rs:5:18
  |
5 |     let result = combinebits!("aaaa aaaa bbbb bbbb");
  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: a local variable with a similar name exists: `a`
  |
  = note: this error originates in the macro `combinebits` (in Nightly builds, run with -Z macro-backtrace for more info)
